### DISCUSSION TITLE
How to drive an hour of data at the site scale

### DISCUSSION THREAD

**Discussion by qiaochenglong** [NCAR/hrldas#124] [Q&A] [answered]
I want to use one or three hours of site data to drive the model. How do I change the time settings?

**Comment by cenlinhe** [comment, marked as answer]
once you generate the 1-hour or 3-hour forcing data, you can change the following entry in the namelist.hrldas:
FORCING_TIMESTEP = 3600 (for hourly forcing) or 10800 (for 3hourly forcing).
you can keep using NOAH_TIMESTEP    = 1800 (half-hourly model timestep) or 3600 (hourly model timestep). If you see water balance issue that leads to model crash, firt try to reduce model time step from 3600 to 1800, or from 1800 to 900.

**Reply by qiaochenglong** [reply]
Thank you very much for your response. There are two issues:
1.Should the time intervals in the forced data be changed to smaller intervals, such as 1998 01 02 00 00; 1998 01 02 01 00; 1998 01 02 02 00?
2.In the sample data, using MODIS LAI data from 2000 to 2008 at a monthly scale, one row for each month, but is each column a year? Why would the monthly scale MODIS LAI data for 9 years have 20 columns?
![Uploadi<img width="1115" alt="屏幕截图 2024-07-24 112621" src="https://github.com/user-attachments/assets/ee773650-e474-4551-9ec7-591c49d61b9c">

**Reply by cenlinhe** [reply]
1. Hourly forcing should be sufficient. You can also use half-hourly like what showed in our point example run.
2. The MODIS LAI is a multi-year monthly average (i.e., climatology) in the parameter table. The 20 columns is for 20 different land types of the LAI climatology. 12 rows mean 12 months.

**Reply by qiaochenglong** [reply]
Dear Dr. He,
In the vegetation dynamics options, options 3 and 4 use "use table LAI," where the LAI is modified in NoahmpTable.TBL. If I want to use options 7-9, which correspond to "use input LAI," how should I input the LAI correctly? This "use input LAI" option is designed for site-scale applications.

**Reply by cenlinhe** [reply]
you need to provide the LAI variables in the hourly forcing file and then choose the option 7-9 will allow the model to read the LAI value from forcing file together with other forcing variables.

**Reply by qiaochenglong** [reply]
Thank you very much for your answer!

**Reply by qiaochenglong** [reply]
You previously mentioned that selecting vegetation dynamics options 7–9 would allow the model to read LAI values and other forcing variables from the forcing file. I would like to confirm: as shown in the attached image, should I input both LAI and FVC? If so, is there anything else I need to do? If not, could you please advise me on the correct procedure?

Additionally, after configuring the settings and compiling the model, I noticed that the generated .nc file does not contain the LAI variable. Furthermore, when I enabled vegetation dynamics option 8, the resulting LAI values were all 0.05. Could you please help me understand what might be causing this issue?

![1](https://github.com/user-attachments/assets/fc57c2a4-e969-4310-b88a-7183498879c6)

**Reply by cenlinhe** [reply]
The current HRLDAS_forcing pre-processor package cannot automatically convert the point-scale LAI and veg fraction vector into the netcdf input file. You will have to generate the regular netcdf forcing file first, and then write your own script to add the hourly LAI and vegetation fraction data into the hourly forcing netcdf file.
For veg option = 8, because your forcing file does not have LAI input, the model automatically assign a minimum value (0.05) for the LAI.
